src/main/common/router/routablePath.ts

```diff
@@ -30,4 +32,21 @@ export class RoutablePath {
     const viewPath: string = split.slice(isCaseUri ? 2 : 1).join('/')
     return `${featureName}/views/${viewPath}`
   }
+
+  evaluateUri (substitutions: {[key: string]: string}): string {
+    if (substitutions == null || Object.keys(substitutions).length === 0) {
```

Comment:
!substitutions? we don't use null in our codebase??

Reply:
I made that deliberately thinking of this as something we could to extract in the future. I tend to use nulls only for places which are generic enough to be extracted in the future.

Reply:
even so other people shouldn't use null either, even if extracted it should still be the same logic I would have thought?

Reply:
Agree, that is why I decided to check for null as well. In first version I only had `substitutions  === undefined` but that would allow nulls for those who did not discard them.

A `substitutions == null` is doing both `substitutions === undefined` and `substitutions === null`. I did not want to use `!substitutions` since the that covers too many cases including `false` or `''`.